src/domains/login/page/KakaoSuccess.jsx

```
@@ -0,0 +1,48 @@
+import { useEffect } from 'react';
```

ESLint

[error] 'useEffect' is defined but never used.

(no-unused-vars)